src/rust/engine/process_execution/src/local.rs

```diff
@@ -216,41 +216,29 @@ impl ChildResults {
 }
 
 impl super::CommandRunner for CommandRunner {
-  ///
-  /// Runs a command on this machine in the passed working directory.
-  ///
-  /// TODO: start to create workunits for local process execution
-  ///
-
-  fn get_platform(&self) -> Platform {
-    self.platform.clone()
-  }
-
-  fn is_compatible_request(&self, req: &MultiPlatformExecuteProcessRequest) -> bool {
-    req.0.contains_key(&(Platform::None, Platform::None))
-      || req
-        .0
-        .contains_key(&(self.platform.clone(), Platform::current_platform().unwrap()))
-  }
-
   fn get_compatible_request(
```

Comment:
Nit: `get_` prefixes on getters aren't common in rust (I don't think)? Maybe `extract_*`, or `compatible_request_from`?